### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by cgwalters**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -44,8 +44,8 @@ func main() {
 	flag.BoolVar(&keep, "keep", false, "Do not remove container image")
 	flag.BoolVar(&reboot, "reboot", false, "reboot if changed")
 	flag.Usage = func() {
-		fmt.Printf("pivot: version %s (%s)\n", version, commitHash)
-		fmt.Printf("Usage:\n")
+		fmt.Println("pivot: version %s (%s)\n", version, commitHash)
```
Comment (ashcrow) [inline main.go]: Why switch to ``fmt.Println()`` *with* newlines?

Comment (cgwalters) [reply main.go]: Yep, good catch, updated  :arrow_down:

**Comment by cgwalters** [comment]
Although there's more format issues here, working on that.

**Comment by cgwalters** [comment]
OK, why the $@$! does `fmt.Println` not take a format string?  Anyways, updated :arrow_up:

**Pull request merged**